backend/djangoindia/db/models/common.py

```diff
@@ -0,0 +1,19 @@
+from .base import BaseModel
+from django.db import models
+
+
+class BackgroundTaskLog(BaseModel):
+    """
+        model to hold the information about the background task that runs.
+    """
+    class StatusChoices(models.TextChoices):
+        SUCCESSFULL = "successfully"
+        FAILURE = "FAILURE"
```

Comment:
I'd recommend using more status choices: Pending, Started. and keep Pending as default status.

There can be multiple tasks in the queue, so we need a status for task that have not "started" to execute and are still in the queue.

Once started, there might be a possibility that server crashed or some response is rejected or anything that didn't allow the task to even run. In that case we should have a record.